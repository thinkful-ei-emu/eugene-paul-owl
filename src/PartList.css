.avatar-part{
  width: 30px;
  height:30px;
  margin-top:5px;
  display:inline-block;
  vertical-align: top;
}

.part-name{
  margin:0;
  display:inline-block;
  font-size:15px;
  line-height:30px;
}

p{
  margin:0;
  line-height:30px;
}

.part-info{
  display:inline-block;
  position:relative;
  margin-left:5px;
  vertical-align:top;
  height:60px;
}

.part-status{
  position:absolute;
  bottom:10px;
  left:0px;
}

.participant{
  height: 70px;
}