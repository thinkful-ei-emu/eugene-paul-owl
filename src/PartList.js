import React from 'react';

function PartList(props){
  let chatList=props.participants.map((participant)=>
    (
    <div className='participant'> 
      <img src={participant.avatar} alt='avatar' className='avatar'/>
      <h3> {participant.name}</h3>
      <p>
        {participant.true}
      </p>
    </div>
    )
  )

  return(
    <div>
      {chatList}
    </div>
  );
}

export default PartList;