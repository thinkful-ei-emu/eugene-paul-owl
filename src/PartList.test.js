import React from 'react';
import ReactDOM from 'react-dom';
import PartList from './PartList';
import renderer from 'react-test-renderer';
import {chatEvents, participants} from './Store';


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<PartList participants={participants} />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('renders UI with default props', () => {
  const tree = renderer.create(<PartList participants={participants} />);
  expect(tree).toMatchSnapshot();
});