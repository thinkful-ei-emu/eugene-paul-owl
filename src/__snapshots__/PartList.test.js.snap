// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI with default props 1`] = `
<div>
  <div
    className="participant"
  >
    <img
      alt="avatar"
      className="avatar"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h3>
       
      Koren Templeton
    </h3>
    <p />
  </div>
  <div
    className="participant"
  >
    <img
      alt="avatar"
      className="avatar"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <h3>
       
      Caty Flucker
    </h3>
    <p />
  </div>
  <div
    className="participant"
  >
    <img
      alt="avatar"
      className="avatar"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <h3>
       
      Axe Kubicka
    </h3>
    <p />
  </div>
  <div
    className="participant"
  >
    <img
      alt="avatar"
      className="avatar"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <h3>
       
      Frank Runciman
    </h3>
    <p />
  </div>
  <div
    className="participant"
  >
    <img
      alt="avatar"
      className="avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <h3>
       
      Ashla Attwool
    </h3>
    <p />
  </div>
</div>
`;
